src/oneNotePicker.tsx

```diff
@@ -64,5 +64,5 @@ export class OneNotePicker extends OneNotePickerBase<OneNotePickerProps, {}> {
 }
 
 export * from './oneNoteSingleNotebookPicker';
-export * from './oneNoteSingleNotebookDropdown';
+export * from './oneNotePickerDropdown';
```

Comment:
Did you mean to remove oneNoteSingleNotebookDropdown?

Reply:
Yes- it is in the oneNotePickerDropdown file

Reply:
I think we should still keep it in (more resilient against refactor). We should explicitly keep all the widgets as exports in this file.

Reply:
As discussed, you're right in doing this :)